topping_bot/cogs/inventory.py

```diff
@@ -294,6 +294,104 @@ async def updateinv(self, ctx):
             ],
         )
 
+
+    @inv.command(aliases=["dt"], brief="Delete topping", description="Delete topping(s) from inventory by index")
+    async def deletetopping(self, ctx, indices=parameter(description="indices of toppings in inventory separated by space (up to 25 toppings)", default=[], converter=IndicesConverter[int])):    
```

Comment:
I wonder if we shouldn't name this `!inv toppingdelete`, just so there's not the case of people partially typing `!inv delete` and wiping their whole inventory. I'll go back and add the remove toppings menu to `!inv delete` to mitigate, but still might prevent pain later on

Reply:
In theory yeah, its just a nested command group. However that makes accessing help information harder iirc, and I'd like not to add a command group unless >= 2 commands are part of it (e.g. !inv & !req). I'm fine with either `toppingdelete` or `deletetopping`, I might just take a quick poll and go with that

Reply:
Okay, `!inv deletetopping` seems to be the clear winner. I still would change the alias to be `!inv delt`. For all the other aliases, `d` means download, `del` means delete.